src/jvm/main/org/jetbrains/kotlinx/lincheck/Utils.kt

```diff
@@ -10,19 +10,21 @@
 package org.jetbrains.kotlinx.lincheck
 
 import kotlinx.coroutines.*
-import sun.nio.ch.lincheck.EventTracker
-import sun.nio.ch.lincheck.TestThread
 import org.jetbrains.kotlinx.lincheck.runner.*
 import org.jetbrains.kotlinx.lincheck.strategy.managed.*
 import org.jetbrains.kotlinx.lincheck.transformation.LincheckClassFileTransformer
 import org.jetbrains.kotlinx.lincheck.util.UnsafeHolder
+import org.jetbrains.kotlinx.lincheck.util.readField
 import org.jetbrains.kotlinx.lincheck.verifier.*
+import sun.nio.ch.lincheck.EventTracker
+import sun.nio.ch.lincheck.TestThread
```

Comment:
Unused imports?

Reply:
They are used in `runInIgnoredSection(...)` method